src/locales/en.json

```diff
@@ -24,6 +24,7 @@
     "LABEL_SETTINGS_DISCORD_SHOW_IDLE": "Show track when playback is paused",
     "LABEL_SETTINGS_DISCORD_SHOW_TIME": "Show track time",
     "LABEL_SETTINGS_DISCORD_SHOW_TITLE": "Show title",
+    "LABEL_SETTINGS_DISCORD_USE_DEFAULT": "Use default",
```

Comment:
"Use alternative layout" - sounds better or word it how you wish.
as mentioned by my review in the `discord_settings.html`